taichi/analysis/verify.cpp

```diff
@@ -41,14 +41,12 @@ class IRVerifier : public BasicStmtVisitor {
           break;
         }
       }
-      TI_ASSERT_INFO(
-          found,
-          "IR broken: stmt {} cannot have operand {}."
-          " Consider adding `ti.core.toggle_advanced_optimization(False)`."
-          " If that fixes the problem, please report this bug by opening an"
-          " issue at https://github.com/taichi-dev/taichi to help us improve."
-          " Thanks!",
-          stmt->id, op->id);
+      TI_ASSERT_INFO(found,
+                     "IR broken: stmt {} cannot have operand {}."
+                     " Please report this bug by opening an issue at"
```

Comment:
```suggestion
                     " If you are using autodiff, please check"
                     " https://taichi.readthedocs.io/en/stable/differentiable_programming.html#kernel-simplicity-rule."
                     " If it doesn't help, please report this bug by opening an issue at"
```

Not sure if we want to add this... Programs violating these rules often result in this error message.

Note that the URL is so long that @taichi-gardener may complain...